CMakeLists.txt

```diff
@@ -74,6 +74,8 @@ if (MSVC)
         message(STATUS "Add /bigobj flag to compiler")
         add_compile_options(/bigobj)
     endif()
+    # For Visual Studio users
+    file(COPY_FILE ${PROJECT_SOURCE_DIR}/CMake/.runsettings ${PROJECT_BINARY_DIR}/.runsettings )
```

Comment:
Need to conditionally copy the file: `if(CMAKE_VERSION VERSION_GREATER_EQUAL 3.21)`

Reply:
Ok I conditioned the call